lib/UserInterface/textinput/StreamReaderWin.cpp

```diff
@@ -176,9 +176,20 @@ namespace textinput {
         return false;
       }
     } else {
+      //https://msdn.microsoft.com/en-us/library/windows/desktop/aa365690(v=vs.85).aspx
```

Comment:
Is this a change, relevant only for windows? Please add two lines explaining in plain English why we need to do that, instead of a link, which might change in a few years from now.